Alexandria/src/main/webapp/books.jsp

```diff
@@ -0,0 +1,91 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```

Comment:
We dont need this jsp anymore do we? Because the books are all located in the book search jsp right?